app/src/main/java/org/simple/clinic/editpatient/EditPatientScreen.kt

```diff
@@ -181,6 +181,17 @@ class EditPatientScreen(context: Context, attributeSet: AttributeSet) : Relative
   }
 
   private fun setInputFieldsHint() {
+    val allGendersRadioButtons = mapOf(
+        Male to maleRadioButton,
+        Female to femaleRadioButton,
+        Transgender to transgenderRadioButton
+    )
+    val genderField = inputFields.fields.find { it is GenderField } as GenderField
+
+    allGendersRadioButtons.forEach { (gender, radioButton) ->
+      radioButton.visibleOrGone(gender in genderField.allowedGenders)
+    }
```

Comment:
This is currently hiding or showing the fields, but the name of the method is `setInputFieldsHint`. The implementation does something which is not clear from the name.

Can we either:
- Rename the method that makes sense for the implementation, or
- Move the hiding or showing of fields to a separate method?

Reply:
Moved the hiding or showing gender radio buttons to `showOrHideGenderRadioButtons` method.